### DISCUSSION TITLE
Group by taxonomy

### DISCUSSION THREAD

**Discussion by relax222** [OrchardCMS/OrchardCore#10529] [Q&A]
Hi guys ! I have a wonder with taxonomy module.
Can I access parent node with all blog post of all child nodes
![image](https://user-images.githubusercontent.com/92966163/138390451-952292ee-4606-400f-98a6-a475d4242cc0.png)
 I have 2 posts Nature category, but i dont find it with Travel

**Comment by ns8482e** [comment]
I guess using TaxonomyIndex you can query  Inherited term
